tests/dumpsmach_test.py

```diff
@@ -64,8 +61,12 @@ def test_nx():
     g.add_edge(1, 2, a=1, b=0, c=1, d=1)
     print(dumpsmach.python_case(g, classname='Machine', start='Sinit'))
     exe_globals = dict()
-    exec(dumpsmach.python_case(g, classname='Machine', start='Sinit'), exe_globals)
-    m = exe_globals['Machine']()  # previous line creates the class `Machine`
+    exec(
+        dumpsmach.python_case(
+            g, classname='Machine', start='Sinit'),
+        exe_globals)
+    m = exe_globals['Machine']()
+        # previous line creates the class `Machine`
```

Comment:
Moving "previous line creates the class `Machine`" to the line after `exe_globals['Machine']()` is confusing because "previous line" refers to the line with `exec()`. I recommend to move it up a line, i.e.,
```suggestion
        exe_globals)
        # previous line creates the class `Machine`
    m = exe_globals['Machine']()
```